src/gameboy/PPU/Sprite.h

```diff
@@ -29,17 +28,13 @@ class Sprite {
 
     bool coversLine(uint8_t line, unsigned int objectSize) const;
     bool containsX(uint8_t lcdX) const;
+    bool hasHigherPriorityThan(const std::shared_ptr<Sprite>& other) const;
 
     uint8_t getTileID(uint8_t lcdY) const;
     uint8_t getTileX(uint8_t lcdX) const;
     uint8_t getTileY(uint8_t lcdY) const;
     uint8_t getPaletteNumber() const;
-
-    bool hasHigherPriorityThan(const std::shared_ptr<Sprite>& other) const;
     bool backgroundOverSprite() const;
 
-    void print();
+    void print() const;
```

Comment:
Why use the `const` keyword when nothing is returned?

Reply:
Oh, I think I just did it since Clion suggested it.